.dotnet/src/Custom/Chat/ChatMessageContentPart.cs

```diff
@@ -146,10 +165,23 @@ public static ChatMessageContentPart CreateRefusalPart(string refusal)
 
         return new ChatMessageContentPart(
             kind: ChatMessageContentPartKind.Refusal,
-            text: null,
-            imageUri: null,
-            refusal: refusal,
-            serializedAdditionalRawData: null);
+            refusal: refusal);
+    }
+
+    /// <summary> Creates a new <see cref="ChatMessageContentPart"/> that encapsulates user role input audio in a known format. </summary>
+    /// <remarks>
+    /// Binary audio content parts may only be used with <see cref="UserChatMessage"/> instances to represent user audio input. When referring to
+    /// past audio output from the model, use <see cref="ChatOutputAudioReference(string)"/> instead.
+    /// </remarks>
+    /// <param name="audioBytes"> The audio data. </param>
+    /// <param name="audioFormat"> The format of the audio data. </param>
+    public static ChatMessageContentPart CreateInputAudioPart(BinaryData audioBytes, ChatInputAudioFormat audioFormat)
```

Comment:
Looking at this method, the `CreateImagePart` method now looks so silly because it takes a plain string for the MIME media type. Ugh... We will deal with that another day. 😄